provision/kubernetes/cluster.go

```diff
@@ -498,6 +500,17 @@ func (c *ClusterClient) SinglePool() (bool, error) {
 	return strconv.ParseBool(singlePool)
 }
 
+func (c *ClusterClient) AvoidMultipleServices() (bool, error) {
```

Comment:
@arthurcgc I did not see a call to `AvoidMultipleServices` bellow, I think that we could use that without flag.

Reply:
That's actually a really nice catch... The right thing to do would be to check if `newVersion` and the clusterFlag is set inside ensureServices... do you agree?

Reply:
yeah @arthurcgc, in the feature we could drop this flag, them turn this a default behavior